### PR TITLE
[D] Various Fixes

### PR DESCRIPTION
**Pull request by BenjaminSchaaf**

### CODE REVIEW

File: D/D.sublime-syntax [hunk 1]
```diff
@@ -1054,29 +1056,29 @@ contexts:
...
           set: [first-value-call-after, value-list]
         - match: '(?=\S)'
-          set: [first-value-after, meta-function-call, value-template-single]
+          set: [first-value-after, meta-function-call-template, value-template-single]
     - match: '\['
       scope: punctuation.section.brackets.begin.d
       set: [first-value-bracket-after, value-list]
     - match: ','
-      scope: punctuation.separator.sequence.d
+      scope: punctuation.separator.sequence.d2
```
Comment (jfcherng) [inline D/D.sublime-syntax:1081]: typo

File: D/D.sublime-syntax [hunk 2]
```diff
@@ -1465,20 +1463,17 @@ contexts:
   value-condition:
     - match: (?=:)
       pop: 1
```
Comment (jrappen) [inline D/D.sublime-syntax:1465]: this `pop: 1` should be `pop: true` as the syntax is v1

**Comment by jrappen** [comment]
Other than that this generally seems fine.

File: D/D.sublime-syntax (hunk 2 above)
Comment (BenjaminSchaaf) [reply D/D.sublime-syntax]: `pop: 1` works in v1 as well and that line wasn't changed in this PR.

**Comment by BenjaminSchaaf** [comment]
Should be ready to merge.

**Review by keith-hall — APPROVED** [review]

**Review by jfcherng — APPROVED** [review]

**Pull request merged**